client/my-sites/domains/domain-management/edit/domain-types/mapped-domain-type.jsx

```diff
@@ -48,10 +51,50 @@ class MappedDomainType extends React.Component {
 				{ this.dnsRecordsNavItem() }
 				{ this.domainConnectMappingNavItem() }
 				{ this.transferMappedDomainNavItem() }
+				{ this.deleteMappingNavItem() }
 			</VerticalNav>
 		);
 	}
 
+	deleteMappingNavItem() {
+		const { domain, isLoadingPurchase, purchase, selectedSite, translate } = this.props;
+
+		if ( ! domain.currentUserCanManage ) {
+			return null;
+		}
+
+		if ( isLoadingPurchase ) {
+			return <VerticalNavItem isPlaceholder />;
+		}
+
+		if ( ! selectedSite || ! purchase ) {
+			return null;
+		}
+
+		const title = translate( 'Delete your domain mapping' );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **34** times:
`translate( 'Delete Domain Mapping' )` *ES Score: 7*